migrate.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE IF NOT EXISTS `vehicle_financing` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `citizenid` varchar(50) DEFAULT NULL,
+    `plate` varchar(15) NOT NULL,
+    `balance` int(11) NOT NULL DEFAULT 0,
+    `paymentamount` int(11) NOT NULL DEFAULT 0,
+    `paymentsleft` int(11) NOT NULL DEFAULT 0,

```

Comment:
Do we need to continue to store payments left? Feels like a derived field of the other 3

Reply:
If you're willing to remove it from the lua why not, i don't want to break it as i can't seem to really understand how the financing system works.